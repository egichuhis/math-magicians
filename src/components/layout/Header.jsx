import React from 'react';
import styles from '../styles/Header.module.css';

const Header = () => (
  <nav className={styles.nav}>
    <h1>Math Magicians</h1>
    <ul>
      <a href="/"><li>Home</li></a>
      <a href="/calculator"><li>Calculator</li></a>
      <a href="/quote"><li>Quote</li></a>
    </ul>
  </nav>
);

export default Header;
