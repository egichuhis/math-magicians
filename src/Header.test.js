import React from 'react';
import { render } from '@testing-library/react';
import Header from './components/layout/Header';

test('Header renders correctly', () => {
  const { container } = render(<Header />);
  expect(container).toMatchSnapshot();
});
