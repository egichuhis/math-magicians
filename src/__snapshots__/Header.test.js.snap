// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div>
  <nav
    class="nav"
  >
    <h1>
      Math Magicians
    </h1>
    <ul>
      <a
        href="/"
      >
        <li>
          Home
        </li>
      </a>
      <a
        href="/calculator"
      >
        <li>
          Calculator
        </li>
      </a>
      <a
        href="/quote"
      >
        <li>
          Quote
        </li>
      </a>
    </ul>
  </nav>
</div>
`;
